src/main/java/com/atlassian/bitbucket/jenkins/internal/jenkins/oauth/consumer/OAuthConsumerUpdateAction.java

```diff
@@ -84,6 +95,25 @@ public String getIconFileName() {
         return "setting.png";
     }
 
+    public String getJenkinsBaseUrl() {
+        return jenkinsProvider.get().getRootUrl();
```

Comment:
Good question. JenkinsProvider is actually an accessor class to the `Jenkins` instance rather than an optional, and this wraps our `Jenkins.get()` call- which _absolutely can return null_ in some circumstances. To my broad understanding though, if the Jenkins instance isn't up, this page is inaccessible anyway so I think we're safe.